src/pykiso/cli.py

```diff
@@ -90,12 +90,34 @@ def check_file_extension(
     return paths
 
 
+class CommandWithOptionalFlagValues(click.Command):
+    def parse_args(self, ctx, args):
+        """Translate any flag `--opt=value` as flag `--opt` with changed flag_value=value"""
+        # filter flags
+        flags = [
+            o
+            for o in self.params
```

Comment:
Please change the naming from o to param. Same for some of the other loops below, it's best to give descriptive names instead of single character names

Reply:
Ok, i will fix it